vault-to-tink/export-contexts.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# NOTES:
+# This script will assume:
+# * lein dependencies are pre-cached OR the internet is accessible
+# * we have modified the kubectl context to be in the correct namespace (kubectl config set-context --current --namespace <namespace>)
+# * Docker is running & can be utilized without sudo
+# * the Postgres server is internal to the CircleCI installation
+
+SVC_NAME=contexts-service
+SVC_PORT=6005
+
+rm -f contexts.json
+touch contexts.json
```

Comment:
```suggestion
echo "" > contexts.json
```

Reply:
This will not work but I can use `echo -n > contexts.json` instead